mousetrap.js

```diff
@@ -1018,4 +1023,4 @@
             return Mousetrap;
         });
     }
-}) (window, document);
+}) (typeof window !== 'undefined' ? window : null, typeof document !== 'undefined' ? document : null);
```

Comment:
Wow, I submitted this by an error, wanted to submit a PR to our corporate fork because honestly, the same PR from another guy just hanged here without any kind of attention since summer 2015.
But heck, if you've noticed it now, I might as well correct it and make this PR happen.
This is going to ease a lot of server-side pain for a lot of people.